.github/workflows/release-please.yml

```diff
@@ -0,0 +1,35 @@
+name: Run Release Please
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  release-please:
+    runs-on: ubuntu-latest
+
+    permissions:
+      id-token: write # Needed for OIDC to get release secrets.
+      contents: write # Contents and pull-requests are for release-please to make releases.
+      pull-requests: write
+
+    outputs:
+      releases_created: ${{ steps.release.outputs.releases_created }}
+
+    steps:
+      - uses: google-github-actions/release-please-action@v4
+        id: release
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          target-branch: ${{ github.ref_name }}
+
+  call-workflow-publish:
+    needs: release-please
+    uses: ./.github/workflows/publish.yml
+    if: ${{ needs.release-please.outputs.releases_created == 'true' }}
+    with:
+      run_tests: true
+#      TODO: revert dry run put in place during testing
```

Comment:
Do you want to change this one now, or after more testing?

Reply:
I'd like to keep it as dry run so that we can see the reviewed code changes run on main.